copter/source/docs/autotune.rst

```diff
@@ -155,10 +157,31 @@ Common Problems
 - If the vehicle will not start tuning (i.e. it won't twitch) even though it is in AutoTune mode then the problem is likely that the roll, pitch, yaw or throttle sticks are not exactly in the middle. It may help to increase the deadzone on the RC input by increasing :ref:`RC1_DZ <RC1_DZ>`, :ref:`RC2_DZ <RC2_DZ>`, :ref:`RC3_DZ <RC3_DZ>` and :ref:`RC4_DZ <RC4_DZ>` to 50 (or higher).
 - If the AutoTune produces an overly twitchy vehicle try reducing the :ref:`AUTOTUNE_AGGR <AUTOTUNE_AGGR>` parameter (should never be below 0.05) and perform the AutoTune again.
 - If the AutoTune produces a sloppy vehicle, try increasing the :ref:`AUTOTUNE_AGGR <AUTOTUNE_AGGR>` parameter (should never be above 0.1) and perform the AutoTune again.
+- If the battery gets depleted before you can complete the Autotune flight(s), download the latest ``.bin`` dataflash log file from the vehicle.
```

Comment:
this is something Leonard has told me explicitly NOT to do! intermediate values can result in instability and a crash...I had something similar in the wiki and he had me remove it